DEVELOPMENT.md

```diff
@@ -808,7 +718,37 @@ We use [Cypress](https://cypress.io) for running frontend-based end-to-end tests
 
 #### Running frontend end-to-end tests locally
 
-Make sure to [build the frontend assets](#building-the-frontend) and run `tools/bin/mage dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs. To run the stack when working on end-to-end tests, use the `tools/bin/mage dev:startDevStack` command. This will run the runs The Things Stack with proper configuration for the end-to-end tests. Note: this command does not output anything, but logs are written to `.cache/devStack.log`.
+The preferred way to run the end-to-end tests is to use the interactive launcher tool with:
+
+```bash
+$  tools/bin/mage dev:serveDevWebui
+```
+
+1. Make sure to run `tools/bin/mage dev:dbStop dev:dbErase dev:dbCreate dev:dbStart dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs.
+2. Run `tools/bin/mage dev:serveDevWebui`
+3. Select "Cypress" in the launcher tool
+4. Add the following environment variables:
+
+```bash
+  export NODE_ENV="development"
+  export CYPRESS_BASE_URL="http://localhost:8080"
+```
+
+4. Run the end-to-end tests (`tools/bin/mage js:cypressInteractive`)
+
+#### Optional: testing in development environment
+
+Make sure to [build the frontend assets](#building-the-frontend) and run `tools/bin/mage dev:dbStop dev:dbErase dev:dbCreate dev:dbStart dev:initStack dev:sqlDump dev:sqlCreateSeedDb` to create a seed database which the tests can reset the database to in between runs.
```

Comment:
`sqlCreateSeedDb` is present here but its not listed on `tools/bin/mage -l`

Reply:
Additionally if you add the `dev:dbCreate` before the `dev:dbStart` is redundant since the seconds is a dependency of the first, meaning that `dbCreate` calls for `dbStart` by itself